OnShift/SendBird-JavaScript
# PL 82 user invite mobile view

# Things Done
- the two user views (when inviting users to an existing chat and creating a new chat) are now actually mobile styled
  - larger hit targets and buttons
  - better image sizes
  - no change to desktop experience 

# How To Test
- bring up the mobile view
- see how it looks when creating a new chat
- see how it looks when inviting members to a new chat
- desktop views mostly unchanged
